src/Semantics.js

```diff
@@ -96,6 +96,30 @@ Object.defineProperties(Wrapper.prototype, {
   // Returns the number of children of this CST node.
   numChildren: {get: function() { return this._node.numChildren(); }},
 
+  // Returns the minimum number of children for this CST node, if it's an iterator node.
+  // Otherwise, throws an exception.
+  minNumChildren: {
+    get: function() {
+      if (this.isIteration()) {
+        return this._node.minMatches;
+      }
+      throw new TypeError(
```

Comment:
The problem is that `minNumChildren` doesn't make sense for anything else than iterator nodes. Any other node has an exact number of children (and an iteration node for optional matches).